src/cloud/project/user-admin.md

```diff
@@ -104,42 +106,77 @@ magento-cloud user:role alice@example.com --level environment --environment deve
 
 {:.procedure}
-To create user accounts from the Project Web UI:
+Access the Project Web UI to add users:
 
-1. Log in to [your {{site.data.var.ece}} account](https://accounts.magento.cloud).
+1. Log in to [your {{site.data.var.ece}} account][].
 
 1. Click the **Projects** tab as the following figure shows.
 
    ![Click the projects tab to access your Cloud project]({{ site.baseurl }}/common/images/cloud/cloud_account_project.png){:width="550px"}
 
-1. Click the name of your project.
+1. Click the name of your project to open the Cloud project portal (Onboarding UI).
+
+1. Click **Infrastructure access**, and then click the **Project Access (Web UI)** link.
+
+   ![Cloud project portal]({{ site.baseurl }}/common/images/cloud/cloud-login-infrastructure-access.png){:width="550px"}
+
+1. In the Project Web UI, add project-level users and environment-level users as needed.
+
+{:.procedure}
+Add a project-level user:
 
-1. Click the configure project button next to project name in the top navigation bar as the following figure shows.
+1. In the Project Web UI, click the configure project gear icon next to project name in the top navigation bar.
 
    ![Configure the project]({{ site.baseurl }}/common/images/cloud/cloud_project_gear.png){:width="184px"}
 
-1. In the right pane, click **Add Users**.
+1. In the _Users_ tab, click **Add User**.
 
    ![Start creating users]({{ site.baseurl }}/common/images/cloud/cloud_project-config.png){:width="500px"}
 
-1. Click **Add User**.
+1. Complete the _Add User_ form:
 
-   ![Add users]({{ site.baseurl }}/common/images/cloud/cloud_project-add-superuser.png){:width="500px"}
+   ![Add users]({{ site.baseurl }}/common/images/cloud/cloud_project-add-super-user.png){:width="500px"}
 
-1. Enter the user e-mail address.
+   -  Enter the user e-mail address.
 
-1. Select the access for the account:
+   -  Select the access for the account:
 
-   -  For a project administrator account, select the **Super User** checkbox. This provides Admin rights for all settings and environments. If not selected, the account has only view options for all project environments.
+      For a project administrator account, select the **Super User** checkbox. This provides Admin rights for all settings and environments. If not selected, the account has only view options for all project environments.
 
    -  Select permissions per specific environment (or branch) in the Integration environment: _No access_, _Admin_ (change settings, execute action, merge code), _Contributor_ (push code), or _Reader_ (view only). When you add active environments, you can modify permissions per user.
 
 1. Click **Add User**.
 
-   {%include cloud/note-prevent-site-availability-issues.md%}
+   {:.bs-callout-warning}
+   After you add project-level users, you must redeploy all environments to apply the changes. To trigger a redeploy, you need to push a new commit to the environment.
```

Comment:
If they are in the Project Web UI, pushing a commit may not make sense. When they add a user in the UI, there is an automatic "something" going on, which doesn't act like a full redeploy but I'm not sure. If they need a full redeploy, then they can use the `redeploy` command.

Reply:
I removed the info on how to redeploy. The main point is that they have to redploy. It doesn't happen automatically as it does when they add an environment user.